src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -11,12 +11,16 @@
 
 namespace Symfony\Component\Console\Helper;
 
+@trigger_error('The '.__NAMESPACE__.'\Helper class is deprecated since version 4.2 and will be removed in 5.0.', E_USER_DEPRECATED);
```

Comment:
core classes still extend this class.. i think this line is problematic. Perhaps keep the soft-deprecation only below, and trigger one on each base method being invoked.